crates/project/src/lsp_store.rs

```diff
@@ -5678,7 +5678,6 @@ impl LspStore {
                     .await
                     .into_iter()
                     .flatten()
-                    .filter(|help| !help.label.is_empty())
```

Comment:
Is it really ok to display empty signatures or what is this removal for?
Can we test it?

Reply:
Ah I see

```
        let label = signature
            .label
            .clone()
            .when(signature.label.is_empty(), |_| "<No Parameters>".into());
```

is this even possible to get?

<img width="580" alt="image" src="https://github.com/user-attachments/assets/60235900-de7e-427c-ac88-672d17f25809" />

How does it look like and why do we need it instead of "no pop-up entry at all"?